.github/workflows/generate.yml

```diff
@@ -0,0 +1,19 @@
+name: Generate CI
+
+on:
+  push:
+    branches:
+      - master
+    schedule:
+      - cron: "0 */5 * * *"
+    workflow_dispatch:
```

Comment:
There's no need to run this on a `cron` schedule nor have a manual trigger for this, or at least I don't think so.

Reply:
Done.